src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+        },
+        "duration_units": {
+            "title": "Time duration units",
+            "type": "string",
+            "enum": [
+                "second",
+                "minute",
+                "hour",
+                "day"
+            ],
+            "comment": "duration_units should be used in conjunction with duration"
+        },
+        "pH": {
+            "title": "pH",
+            "type": "number",
+            "description": "Final pH of the solution containing a chemical compund (if applicable)"
+        },
+        "purpose": {
+            "title": "Purpose",
+            "type": "string",
+            "enum": [
+                "activation",
+                "agonist",
+                "antagonist",
+                "control",
+                "differentiation",
+                "de-differentiation",
+                "perturbation",
+                "selection",
+                "stimulation"
+            ],
+            "description": "The intended purpose for treating the samples; Activation: treatment is known to activate a pathway in the biosample; Agonist: a substance which is known to initiate a physiological response when combined with a receptor; Antagonist: a substance that is known to interfere with or inhibits the physiological action of another; Control: treatment applied to a sample for control purposes; Differentiation: treatment that is applied to convert a less specialized cell to a more specialized cell; De-differentiation: treatment used to reprogram differentiated cells back to less determined cell states; Perturbation: treatment applied to the sample in order to study the effect of its application; Selection: treatment used to affect biosample in a way that can be used to distinguish cells and select for in the downstream steps; Stimulation: treatment applied to stimulate a cellular pathway."
+        },
+        "post_treatment_time": {
+            "title": "Post-treatment time",
+            "description": "Post treatment time in conjunction with post treatment time units is used to specify the time that has passed between the point when biosamples were removed from the treatment solution before being sampled or treated with the next treatment.",
+            "type": "number",
+            "comment": "post_treatment_time should be used in conjunction with post_treatment_time_units."
+        },
+        "post_treatment_time_units": {
+            "title": "Post-treatment time units",
+            "type": "string",
+            "enum": [
+                "minute",
+                "hour",
+                "day",
+                "week",
+                "month"
+            ],
+            "comment": "post_treatment_time_units should be used in conjunction with post_treatment_time."
+        },
+        "temperature (Celsius)": {
+            "title": "Temperature",
+            "type": "number",
+            "description": "Temperature can only be specified in Celsius"
```

Comment:
Line 180 is an example of mixed description and comment.  Below I have split the concepts.
"description":"The temperature in Celsius to which the sample was exposed"
"comment":"Temperature must be converted to Celsius for submission."